File: Modul 1/PRAK110-2610817210025-MuhammadAzmi.c
#include <stdio.h>
#include <math.h>

int main() {
    int Base = 5;
    int Height = 12;
    int Hypotenuse = (int)round(sqrt(pow(Base, 2) + pow(Height, 2)));
    int Perimeter = Base + Hypotenuse + Height;
    int Area = (Base * Height) / 2;
    
    printf("Diketahui :\n");
    printf("Alas = %d cm\n", Base);
    printf("Tinggi = %d cm\n\n", Height);
    printf("Jawaban :\n");
    printf("Sisi A = %d cm\n", Base);
    printf("Sisi B = %d cm\n", Hypotenuse);
    printf("Sisi C = %d cm\n", Height);
    printf("Keliling = %d cm\n", Perimeter);
    printf("Luas = %d cm\n", Area); 
    return 0;
}

File: Modul 1/PRAK110-2610817210025-MuhammadAzmi.py
import math
Base = 5
Height = 12
Hypotenuse = int(math.sqrt(Base**2 + Height**2))

Perimeter = Base + Hypotenuse + Height
Area = int((Base * Height) / 2)

print("Diketahui :")
print(f"Alas = {Base} cm")
print(f"Tinggi = {Height} cm\n")
print("Jawaban :")
print(f"Sisi A = {Base} cm")
print(f"Sisi B = {Hypotenuse} cm")
print(f"Sisi C = {Height} cm")
print(f"Keliling = {Perimeter} cm")
print(f"Luas = {Area} cm")